viewer-prototype/src/browser/trace-viewer/components/utils/filtrer-tree/indent.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+type IndentProps = {
+    level: number;
+    isLastLevel: boolean;
+}
+
+export class Indent extends React.Component<IndentProps> {
+    constructor(props: IndentProps) {
+        super(props);
+    }
+
+    render() {
+        if (!this.props.level) return null;
+        
+        let { level } = this.props;
+        level = this.props.isLastLevel ? level+1 : level;
```

Comment:
Why a +1 for the last level?

Reply:
to have an extra indent since there is no arrow icon